frontend/src/views/teacher/quizzes/QuizList.vue

```diff
@@ -295,6 +322,20 @@ export default class QuizList extends Vue {
     await this.$store.dispatch('clearLoading');
   }
 
+  async showQuizFraudScores(quiz: Quiz) {
+    await this.$store.dispatch('loading');
+    try {
```

Comment:
if this call is inside the dialog this component would be smaller and the call would be local to where the data is presented. would it be possible?